src/Output/Copy/Request/AuthenticationCredentials.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoclerLabs\ApiClientGenerator\Output\Copy\Request;
+
+class AuthenticationCredentials
+{
+    /** @var string */
+    private string $username;
```

Comment:
anything inside this /Copy/ needs to be compatible with php 7.0 as well, so no property type hints :(